demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -524,6 +524,9 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
         LogLevel.INFO
       );
     }
+    (document.getElementById(
```

Comment:
So I tried this and it got override later and show the client meeting Id instead.

Reply:
yeah, so i had to do an append operation on the line that overwrites it
as HTMLSpanElement).innerHTML += `${this.meeting} (${this.region})`;